docs/siem/timeline/api/timeline-api-create.asciidoc

```diff
@@ -0,0 +1,360 @@
+      },
+      {
+        "id": "event.action"
+      },
+      {
+        "id": "host.name"
+      }
+    ],
+    "dataProviders": [ <1>
+      {
+        "and": [
+          {
+            "name": "event.category",
+            "enabled": true,
+            "excluded": false,
+            "queryMatch": {
+              "field": "event.category",
+              "value": "process",
+              "operator": ":"
+            }
+          },
+          {
+            "name": "user.name",
+            "enabled": true,
+            "excluded": false,
+            "queryMatch": {
+              "field": "user.name",
+              "value": "SYSTEM",
+              "operator": ":"
+            }
+          }
+        ],
+        "enabled": true,
+        "excluded": false,
+        "name": "host.os.platform",
+        "queryMatch": {
+          "field": "host.os.platform",
+          "value": "windows",
+          "operator": ":"
+        }
+      }
+    ],
+    "dateRange": {
+      "end": 1594005719000,
+      "start": 1593832919000
+    },
+    "description": "Gets Windows system processes from all hosts",
+    "title": "Windows system processes"
+  }
+}
+--------------------------------------------------
+// KIBANA
+
+<1> To ensure the timeline is displayed correctly in the UI, specify these
+fields in all `dataProviders` objects:
+
+* `and` (can be empty)
+* `name`
+* `enabled`
+* `excluded`
+* `queryMatch`
+** `field`
+** `value`
+** `operator`
+
+*Example 2*
+
+Creates a new timeline template:
+
+[source,console]
+--------------------------------------------------
+POST api/timeline
+{
+  "timeline": {
+    "columns": [
+      {
+        "id": "@timestamp"
+      },
+      {
+        "id": "user.name"
+      },
+      {
+        "id": "event.category"
+      },
+      {
+        "id": "event.action"
+      },
+      {
+        "id": "host.name"
+      }
+    ],
+    "dataProviders": [
+      {
+        "and": [
+          {
+            "name": "event.category",
+            "enabled": true,
+            "excluded": false,
+            "queryMatch": {
+              "field": "event.category",
+              "operator": ":",
+              "value": "process" <1>
+            }
+          },
+          {
+            "name": "user.name",
+            "enabled": true,
+            "excluded": false,
+            "queryMatch": {
+              "field": "user.name",
+              "operator": ":",
+              "value": "SYSTEM"
+            }
+          }
+        ],
+        "enabled": true,
+        "excluded": false,
+        "name": "host.os.platform",
+        "queryMatch": {
+          "field": "host.os.platform",
+          "operator": ":",
+          "value": "windows"
+        }
+      }
+    ],
+    "dateRange": {
+      "end": 1594005719000,
+      "start": 1593832919000
+    },
+    "description": "Template for investigating host events",
+    "timelineType": "template", <2>
+    "title": "Host event template"
```

Comment:
I think it would be better to also let users know that they call also provide a uuid as `templateTimelineId`, by doing this they can easily update the template by re-importing the template without looking up what templateTimelineId is assigned (as if they don't provide one, we'll assign one for them)

Also I would recommend to put `templateTimelineVersion` as `1` when users creating them, so they would aware that if they want to update the template by re-importing the file, they have to bump the version to any positive integer that's greater than that.